### PR TITLE
Auto line height

### PR DESCRIPTION
**Pull request by Tuizi**
LineHeight is now optionnal + Bump version to 1.1

### CODE REVIEW

File: dist/truncate.js [hunk 1]
```diff
@@ -362,11 +362,23 @@
...
+      lineHeight: 'auto'
     };
 
     this.options = $.extend({}, this._defaults, options);
 
+    if(this.options.lineHeight === 'auto') {
+      var lineHeightCss = this.$element.css('line-height'),
+        lineHeight = 18; // for Normal we return the default value: 18px
+
+      if(lineHeightCss !== "normal") {
```
Comment (jeffchan) [inline dist/truncate.js]: nit: this should be in the linter, but space after `if`

File: dist/truncate.js [hunk 2]
```diff
@@ -362,11 +362,23 @@
...
       showMore: '',
       showLess: '',
-      position: 'end'
+      position: 'end',
+      lineHeight: 'auto'
     };
 
     this.options = $.extend({}, this._defaults, options);
 
+    if(this.options.lineHeight === 'auto') {
```
Comment (jeffchan) [inline dist/truncate.js]: nit: this should be in the linter, but space after `if`

File: dist/truncate.js [hunk 3]
```diff
@@ -362,11 +362,23 @@
...
+      var lineHeightCss = this.$element.css('line-height'),
```
Comment (jeffchan) [inline dist/truncate.js]: Does this work for inherited values?

File: dist/truncate.js (hunk 1 above)
Comment (Tuizi) [inline dist/truncate.js]: Ok, and I will check the linter also

File: dist/truncate.js (hunk 2 above)
Comment (Tuizi) [inline dist/truncate.js]: Ok, and I will check the linter also

File: dist/truncate.js (hunk 3 above)
Comment (Tuizi) [inline dist/truncate.js]: Yes.

```html
<div class="container" style="line-height:30px;max-width:400px;">
  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. </p>
</div>
```

```javascript
$("p").css("line-height"); //output 30px
```

and

```html
<div class="container" style="line-height:30px;max-width:400px;">
  <p style="line-height:inherit">Lorem ipsum dolor sit amet. </p>
</div>
```

```javascript
$("p").css("line-height"); //output 30px
```

Comment (Tuizi) [inline dist/truncate.js]: @jeffchan is it ok for you? :)

File: dist/truncate.js (hunk 2 above)
Comment (Tuizi) [inline dist/truncate.js]: @jeffchan I have added the space in the code. this option doesn't exist in jshint... maybe in eslint

**Comment by jeffchan** [comment]
:+1:

**Pull request merged**